server/app/auth/CiviFormProfileData.java

```diff
@@ -17,6 +18,8 @@
  */
 public class CiviFormProfileData extends CommonProfile {
 
+  private JWT idToken;
+
```

Comment:
**suggestion (blocking):** if this will sometimes be set and sometimes not make it `Optional<JTW>`

Reply:
Done.